### ISSUE TITLE
RLS: 2.2.0

### ISSUE THREAD

**Issue by lithomas1** [pandas-dev/pandas#54902]
Tentative schedule:

RC: 12/22/2023
Actual release: 1/10/2024

blockers
- https://github.com/pandas-dev/pandas/pull/55739

nice to have:
- https://github.com/pandas-dev/pandas/pull/56146

**Comment by lithomas1** [comment]
@pandas-dev/pandas-core 

I'm thinking about cutting a release for the 2.2.0 RC next next Wednesday (the 6th of December).
(I would release this simultaneously with 2.1.4)

This sets us up nicely for an actual release on the 20 of December before Christmas/New Years.

Any thoughts on this date?

**Comment by WillAyd** [comment]
Seems reasonable to me

**Comment by phofl** [comment]
I would prefer not to release directly before Christmas, rather first week of January. 

We need some more warnings for CoW as well

**Comment by jbrockmendel** [comment]
There are a bunch of deprecations id like to get in for 2.2 as well

**Comment by mroeschke** [comment]
I would be OK postponing, but It would be good to list in the OP what items people ideally want to get in before an RC

**Comment by lithomas1** [comment]
I'm fine with a delay, looks like numpy is pushing back their 2.0 release anyways.
Do we want to just cut the RC in january then? 
I don't have time to release the week of Dec 11-15.

Also, it'd be helpful if people edit the OP to list blockers or things they'd like to get in?

**Comment by phofl** [comment]
I'd be fine with cutting the rc end of December if that's possible for you @lithomas1, no worries if not, otherwise first week of January?

**Comment by lithomas1** [comment]
Sure, does anytime during the week of Dec 18-22 work for you?

**Comment by MarcoGorelli** [comment]
@natmokval if there's any deprecations you'd like to get in before 2.2, your input would be valued (including if this timeline looks good)!

**Comment by phofl** [comment]
That should work for me, yes. And then cut the actual release 8th-12th of January?

**Comment by lithomas1** [comment]
Sure, I'll updated the OP with the tentative timeline then.

If anyone else has any concerns about the date, feel free to comment below. 

I'm pretty free from mid December to mid January, so am pretty flexible about moving the release date.

**Comment by jbrockmendel** [comment]
Looking at my list of deprecations I decided I _want_ many of them but none of them are blockers.

**Comment by lithomas1** [comment]
@pandas-dev/pandas-core 

friendly reminder that I'm planning to release the RC tmrw.

If anyone has anything they really like to get in before the RC, now would be the time to comment.
(if the RC doesn't happen tmrw, will just release in January)

**Comment by MarcoGorelli** [comment]
https://github.com/pandas-dev/pandas/pull/56346 is essentially the last piece of the "offset aliases overhaul" puzzle - I think the whatsnew note can be improved, but other than that it looks good

OK to merge in time for the RC, with the understanding that the whatsnew note will be improved in time for the final 2.2 release?

**Comment by phofl** [comment]
yes

**Comment by MarcoGorelli** [comment]
thanks - just going out for lunch but will merge in the next few hours if it's fine

not a blocker anyway, don't feel like you need to wait for it

**Comment by MarcoGorelli** [comment]
> thanks - just going out for lunch but will merge in the next few hours if it's fine

there's some things I'm not sure about in there, so let's not rush things through - it can be deprecated as part of the 3.x cycle, no big deal

So, feel free to release the RC, doesn't look like others have objections either 🚀

**Comment by lithomas1** [comment]
@pandas-dev/pandas-core

Starting the RC release in ~10 minutes, please don't merge anything to main during this time.

**Comment by lithomas1** [comment]
@pandas-dev/pandas-core 

All clear! 
Release is close to finished now (just waiting on conda-forge stuff)

**Comment by lithomas1** [comment]
@pandas-dev/pandas-core 

Starting the release now, please don't merge anything to 2.2.x anymore.

**Comment by lithomas1** [comment]
Just a quick update on the release:

The Windows builds just failed because of the difference in newlines on Windows ('\r\n') vs Unix('\n) for the test for the deprecationwarning for missing pyarrow.

The fix shouldn't be too hard, so I'm still going to try to release today.

**Comment by lithomas1** [comment]
release was announced.

**Comment by datapythonista** [comment]
Thanks for another release @lithomas1. Do you want to create an issue to track the 3.0 release? I can do it myself, but probably worth adding when you expect to release it.

**Comment by lithomas1** [comment]
Feel free to do so.

I'm not too sure when it's possible to release it (I guess April/Mayish).
